docs/src/main/asciidoc/openshift-s2i-guide.adoc

```diff
@@ -0,0 +1,51 @@
+include::./attributes.adoc[]
+= {project-name} - Deploying on OpenShift with S2I
+
+This guide covers:
+
+* The deployment of the application to OpenShift using S2I to build
+
+== Prerequisites
+
+For this guide you need:
+
+* roughly 10 minutes
+* having access to an OpenShift cluster. Minishift is a valid option.
+
+== Solution
+
+We recommend to follow the instructions in the next sections and build the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
```

Comment:
This (and the next paragraph) should be removed, given that I wrote "You do not need to locally clone the Git repository, as it will be directly built since OpenShift." below, agreed @tqvarnst @siamaksade?